crates/bevy_app/src/system_registry.rs

```diff
@@ -0,0 +1,33 @@
+use crate::App;
+use bevy_ecs::prelude::*;
+use bevy_ecs::system::{Callback, SystemRegistryError};
+
+impl App {
+    /// Register a system with any number of [`SystemLabel`]s.
```

Comment:
SystemLabel should be removed from the docs.

Reply:
Yes, the docs are not up to date, I want to stabilize some things first :)